pkg/tsdb/v3iotsdb.go

```diff
@@ -476,12 +476,13 @@ func deleteEntirePartition(logger logger.Logger, container v3io.Container, parti
 	defer wg.Done()
 
 	err := utils.DeleteTable(logger, container, partitionPath, "", workers)
-	if err != nil {
+	if err != nil && !utils.IsNotExistsError(err) {
```

Comment:
move this check to my other comment 😄

Reply:
I would, except it would break the code :)